packages/terra-application-layout/translations/en-CA.json

```diff
@@ -0,0 +1,14 @@
+{
+  "Terra.applicationLayout.applicationHeader.menuToggleLabel": "Toggle Menu",
+  "Terra.applicationLayout.utilityDefaults.menu": "Menu",
+  "Terra.applicationLayout.utilityDefaults.userInformation": "User Information",
+  "Terra.applicationLayout.utilityDefaults.changePhoto": "Change Photo",
+  "Terra.applicationLayout.utilityDefaults.settings": "Settings",
+  "Terra.applicationLayout.utilityDefaults.appearance": "Appearance",
+  "Terra.applicationLayout.utilityDefaults.security": "Security",
+  "Terra.applicationLayout.utilityDefaults.help": "Help",
+  "Terra.applicationLayout.utilityDefaults.gettingStarted": "Getting Started",
+  "Terra.applicationLayout.utilityDefaults.about": "About",
```

Comment:
Pretty sure this should be "Aboot"

Reply:
Are you serious? https://www.atlasobscura.com/articles/whats-going-on-with-the-way-canadians-say-about

Reply:
most definitely not serious